include/boost/url/rfc/detail/hier_part_rule.hpp

```diff
@@ -39,9 +39,8 @@ struct hier_part_rule_t
     {
         bool has_authority = false;
         authority_view authority;
-
-        // VFALCO This doesn't belong here
-        grammar::range<pct_encoded_view> path;
+        grammar::range<
```

Comment:
Are we going to change this to `segments` in the future?

Reply:
I don't know, because this is not user-facing. So there is no real incentive to change it right before the review. We could in theory tidy it up later.